### PR TITLE
Standardizing wins page

### LINKED ISSUE #1166: Standardize Wins Page
### Overview
We need to standardize the Wins page to be more consistent with other pages on the HfLA site

### Action Items
- [x] Remove all CSS formatting from the heading ("Let's Celebrate Together!"). The h1 tag should already automatically give it the correct formatting
- [x] Add margin below the "Let's Celebrate Together!" heading- `margin-bottom: 56px` (in nested h1 class)
- [x] Header image: adjust the max-height of `.wins-hero` `img` class to 360px
- [x] Adjust color of `.wins__see-more-div` from red to $color-red (the HfLA pink color)

### Resources/Instructions
[Wins page](https://www.hackforla.org/wins)

Before:
Screen Shot 2021-03-06 at 9 48 47 PM
<br>
After:
Screen Shot 2021-03-06 at 9 48 40 PM
<br>
Before & after (red to pink):
<br>
Screen Shot 2021-03-06 at 9 48 12 PMScreen Shot 2021-03-06 at 9 48 31 PM

### PR DESCRIPTION
**Pull request by evancchen**
Fixes #1166

### CODE REVIEW

**Comment by Lol-Whut** [comment]
It looks like element nesting got shuffled around a little bit during edits. When I pulled the branch to my machine and accepted incoming changes, I got syntactical error warnings for the curly braces on lines 56 and 87, and styling changes did not show up for me in the browser. @akibrhast are you seeing the same, or is this on my end? 

For reference, right here it looks like there's an extra curly brace:
image

**Review by akibrhast — CHANGES_REQUESTED** [review]
@evancchen 
- Please fix the merge conflicts.
- Please ensures scss nests are properly nested and you can build the changes locally after addressing the merge conflicts

This is what I am seeing on my end when I try to build with `docker-compose up`
```bash
hfla_site    | jekyll 3.9.0 | Error:  Invalid CSS after "...b67d488000a5e71": expected "{", was "}" on line 59
hfla_site    |   Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/main.scss':
hfla_site    |                     Invalid CSS after "...b67d488000a5e71": expected "{", was "}" on line 59
hfla_site    | /usr/gem/gems/jekyll-sass-converter-1.5.2/lib/jekyll/converters/scss.rb:123:in `rescue in convert': Invalid CSS after "...b67d488000a5e71": expected 
"{", was "}" on line 59 (Jekyll::Converters::Scss::SyntaxError)
```

**Comment by akibrhast** [comment]
@Lol-Whut It seems like @evancchen has updated the PR since our last review. Please re-review this and approve his changes if everything looks good , and nothing breaks.

**Comment by Lol-Whut** [comment]
This is definitely looking better! No merge problems, the h1 tag is giving the "Let's Celebrate Together!" its styling and the margin-bottom is now 56px. However, it looks like the max-height of the .wins-hero img class still needs to be adjusted to 360px and the color of .wins_see-more-div still needs to be adjusted to $color-red. I think once those changes are in, it'll be perfect!

I think we should be seeing those two changes in these areas:
image

**Review by Lol-Whut — CHANGES_REQUESTED** [review]
I think once the max-height and color changes I mentioned in my most recent comment are made, we'll be good to go! @evancchen

**Review by akibrhast — APPROVED** [review]

File: _sass/components/_wins-page.scss [hunk 1]
```diff
@@ -6,14 +6,10 @@
     align-items: center;
     justify-content: space-evenly;
     img{
-        height: 366;
+        height: 360px;
```
Comment (Lol-Whut) [inline _sass/components/_wins-page.scss]: change this to max-height and it'll be perf!

**Review by Lol-Whut — CHANGES_REQUESTED** [review]

**Comment by Lol-Whut** [comment]
Looks good to me!

**Pull request merged**